src/index.js

```diff
@@ -208,7 +208,11 @@ export function configureForm ({
 			}
 
 			_createInputOnChange = (name: string, options: ChangeOptions) => (event: any) => {
-				this.updateField(name, event.currentTarget.value, options);
+				if (event.currentTarget.files) {
+					this.updateField(name, event.currentTarget.files[0], options);
```

Comment:
It would be nice to support `multiple` files or at least, not make it hard to add that functionality later.
Any thoughts?

Reply:
Instead of just looking at the value in the onChange, we could actually look at the field configuration that was specified by the user.

It'd look something more like:

```javascript
if (field.type === 'file' && field.multiple) {
  this.updateField(name, event.currentTarget.files, options);
} else if (field.type === 'file') {
  this.updateField(name, event.currentTarget.files[0], options);
} else {
  this.updateField(name, event.currentTarget.value, options);
}
```

Would that make more sense?

Reply:
Yeah I think that’s the way we’d have to do it. I’d like to be able to use this ->

>  If the user selected multiple files, the `value` represents the first file in the list of files they selected. The other files can be identified using the input’s `HTMLInputElement.files` property.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file

Theoretically we could present both the `value` and the `files` attributes but because `FormData` just gives one value per field there’s nowhere to put it.